### PR TITLE
Switching back to main

### PR DESCRIPTION
**Pull request by radoslawc**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
After R3 has it's own branch it's time to switch main to work on main again
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: e2e/tests/oai/002-database.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
   upstream:
     repo: oai-core-packages
     package: database
-    revision: r3
```
Comment (radoslawc) [inline e2e/tests/oai/002-database.yaml:19]: @arora-sagar can this also be main? Previously it was r2 and I was wondering if that's what we want.

File: e2e/tests/oai/002-operators.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ spec:
   upstream:
     repo: oai-core-packages
     package: oai-cp-operators
-    revision: r3
+    revision: main
```
Comment (radoslawc) [inline e2e/tests/oai/002-operators.yaml:19]: @arora-sagar same here, can this be main?

File: e2e/tests/oai/002-database.yaml (hunk 1 above)
Comment (arora-sagar) [reply e2e/tests/oai/002-database.yaml:19]: Yeah sure let me merge

File: e2e/provision/playbooks/roles/install/defaults/main.yml [hunk 3]
```diff
@@ -43,7 +43,7 @@ nephio_stock_repos:
   - name: oai-core-packages
     repo: https://github.com/OPENAIRINTERFACE/oai-packages.git
     directory: /
-    branch: r3
+    branch: main
```
Comment (arora-sagar) [inline e2e/provision/playbooks/roles/install/defaults/main.yml:46]: @efiacor @radoslawc Done --> https://github.com/OPENAIRINTERFACE/oai-packages updated

**Pull request merged**